### ISSUE TITLE
Failed to initialize Vulkan after update. Ubuntu 18.04

### ISSUE THREAD

**Issue by randolphpebenito** [ValveSoftware/Dota-2#1871]
<details><summary> My System Information</summary><p>

```
Computer Information:
    Manufacturer:  Unknown
    Model:  Unknown
    No Touch Input Detected

Processor Information:
    CPU Vendor:  GenuineIntel
    CPU Brand:  Intel(R) Core(TM) i5-4210U CPU @ 1.70GHz
    CPU Family:  0x6
    CPU Model:  0x45
    CPU Stepping:  0x1
    CPU Type:  0x0
    Speed:  2700 Mhz
    4 logical processors
    2 physical processors
    HyperThreading:  Supported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Unsupported
    SSE41:  Supported
    SSE42:  Supported
    AES:  Supported
    AVX:  Supported
    AVX2:  Unsupported
    AVX512F:  Unsupported
    AVX512PF:  Unsupported
    AVX512ER:  Unsupported
    AVX512CD:  Unsupported
    AVX512VNNI:  Unsupported
    SHA:  Unsupported
    CMPXCHG16B:  Supported
    LAHF/SAHF:  Supported
    PrefetchW:  Unsupported

Operating System Version:
    Ubuntu 18.04.2 LTS (64 bit)
    Kernel Name:  Linux
    Kernel Version:  5.4.0-62-generic
    X Server Vendor:  The X.Org Foundation
    X Server Release:  12008000
    X Window Manager:  GNOME Shell
    Steam Runtime Version:  steam-runtime_0.20201203.1

Video Card:
    Driver:  Intel Open Source Technology Center Mesa DRI Intel(R) Haswell Mobile x86/MMX/SSE2
    Driver Version:  3.0 Mesa 19.2.8
    OpenGL Version: 3.0
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 120 Hz
    VendorID:  0x8086
    DeviceID:  0xa16
    Revision Not Detected
    Number of Monitors:  1
    Number of Logical Video Cards:  1
    Primary Display Resolution:  1920 x 1080
    Desktop Resolution: 1920 x 1080
    Primary Display Size: 12.17" x 6.89" (13.98" diag)
                                            30.9cm x 17.5cm (35.5cm diag)
    Primary VRAM Not Detected

Sound card:
    Audio device: Intel Haswell HDMI

Memory:
    RAM:  15925 Mb

VR Hardware:
    VR Headset: None detected

Miscellaneous:
    UI Language:  English
    LANG:  en_PH.UTF-8
    Total Hard Disk Space Available:  702186 Mb
    Largest Free Hard Disk Block:  480546 Mb
```

</p></details>

* Have you checked for system updates?: Yes
* Are you using the latest stable video driver available for your system? [Yes/No]: Yes
* Have you verified the game files?: [Yes/No]: Yes

#### Please describe your issue in as much detail as possible:
Been playing dota 2 since January 2020, then I got an update in Dota 2 yesterday January 20, 2021.
Then upon running, it gives the error "Failed to initialize Vulkan"

![image](https://user-images.githubusercontent.com/5988274/105340283-a3ae1f00-5c18-11eb-893a-a852eab66c2e.png)

Then i tried running the dota2 via terminal to check the logs using the command 
Command: ./dota2.sh
Logs:
```
Using breakpad crash handler
[S_API] SteamAPI_Init(): Loaded '/home/dunadee/.local/share/Steam/linux64/steamclient.so' OK.
Setting breakpad minidump AppID = 570
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
SteamInternal_SetMinidumpSteamID:  Caching Steam ID:  76561199053142417 [API loaded yes]
SteamInternal_SetMinidumpSteamID:  Setting Steam ID:  76561199053142417
Setting breakpad minidump AppID = 373300
 failed to dlopen "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/bin/linuxsteamrt64/librendersystemgl.so" error=libudev.so.0: cannot open shared object file: No such file or directory
 failed to dlopen "librendersystemgl.so" error=libudev.so.0: cannot open shared object file: No such file or directory
```
I tried providing the following:

- Added `-safe_mode` in the launch options (removed -vulkan) **FAILED**
- Added `-vulkan_disable_steam_shader_cache` in the launch options **FAILED**
- Removed/Deleted files inside the shadercache/570  **FAILED**
- Rebooted my computer. **FAILED**
- Uninstall and Install Dota 2. **FAILED**

#### Steps for reproducing this issue:
N/A

**I'm out of options. Hopefully someone can help me :( Willing to pay if necessary.**

**Comment by kisak-valve** [comment]
Hello @randolphpebenito, Dota 2 is not intended to be run from outside the Steam. If you want tinker from the command line, please use something like `~/.local/share/Steam/ubuntu12_32/steam-runtime/run.sh ./dota.sh` to give the game a similar environment as when it's launched from Steam.

Can you check if the `mesa-vulkan-drivers` is installed with something like `apt policy mesa-vulkan-drivers`? Also, it may be worthwhile testing a newer version of mesa from a PPA like [oibaf](https://launchpad.net/~oibaf/+archive/ubuntu/graphics-drivers) or [kisak-mesa](https://launchpad.net/~kisak/+archive/ubuntu/kisak-mesa).

**Comment by danginsburg** [comment]
Also, the most recent update of Dota defaults to Vulkan and will attempt to automatically fallback to OpenGL.  It's failing to fallback to OpenGL because of this:

```
failed to dlopen "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/bin/linuxsteamrt64/librendersystemgl.so" error=libudev.so.0: cannot open shared object file: No such file or directory
 failed to dlopen "librendersystemgl.so" error=libudev.so.0: cannot open shared object file: No such file or directory
```

I've seen this before but forget why it happens, usually people symlink libudev.so.0 to libudev.so.1 I think.

Update: Oh, that's because you're not using run.sh.  Yeah, what happens when you use run.sh?  Can you run with -con_logfile and post the contents of game/dota/console.log?

**Comment by randolphpebenito** [comment]
@kisak-valve @danginsburg 

Executed the `~/.local/share/Steam/ubuntu12_32/steam-runtime/run.sh ./dota.sh` and the dota is now running. 

But now, the problem is, it's not full screen anymore. 
![image](https://user-images.githubusercontent.com/5988274/105436403-190e0400-5c9a-11eb-9469-f6897d933177.png)


![image](https://user-images.githubusercontent.com/5988274/105436488-4064d100-5c9a-11eb-911c-be425cf03e22.png)

**Long List**

<details><summary>Terminal spew</summary><p>

 ```
dunadee@ubuntu-machine  ~/.local/share/Steam/steamapps/common/dota 2 beta/game  ```
~/.local/share/Steam/ubuntu12_32/steam-runtime/run.sh  ./dota.sh -con_logfile                                         ✔
Using breakpad crash handler
[S_API] SteamAPI_Init(): Loaded '/home/dunadee/.local/share/Steam/linux64/steamclient.so' OK.
Setting breakpad minidump AppID = 570
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
SteamInternal_SetMinidumpSteamID:  Caching Steam ID:  76561199053142417 [API loaded yes]
SteamInternal_SetMinidumpSteamID:  Setting Steam ID:  76561199053142417
Setting breakpad minidump AppID = 373300
Attempting to create OpenGL context: v3.3 (Core profile)...success. Checking entry points...success.
Attempting to create OpenGL context: v3.3 (Core profile)...success. Checking entry points...success.
DumpContextInfo: OpenGL vendor Intel Open Source Technology Center
DumpContextInfo: OpenGL renderer Mesa DRI Intel(R) Haswell Mobile 
DumpContextInfo: Using OpenGL context version 4.5
DumpContextInfo: Context supports GLSL version 4.50
Fontconfig warning: "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/core/panorama/fonts/conf.d/41-repl-os-win.conf", line 148: Having multiple values in <test> isn't supported and may 
not work as expected
Fontconfig warning: "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/core/panorama/fonts/conf.d/41-repl-os-win.conf", line 160: Having multiple values in <test> isn't supported and may 
not work as expected
 failed to dlopen "cairo.so" error=cairo.so: cannot open shared object file: No such file or directory
 failed to dlopen "libtensorflow.so" error=libtensorflow.so: cannot open shared object file: No such file or directory
```

</p></details>

Posting the log file with  `~/.local/share/Steam/ubuntu12_32/steam-runtime/run.sh ./dota.sh -con_logfile`

<details><summary>steam_dota_dir/game/dota/console.log output </summary><p>

```
01/22 10:11:44 WARNING: CDirWatcher not implemented
01/22 10:11:44 [RenderSystem] GL_VERSION=4.5 (Core Profile) Mesa 19.2.8
01/22 10:11:44 [RenderSystem] GL_NVX_gpu_memory_info: UNAVAILABLE
01/22 10:11:44 [RenderSystem] GL_ATI_meminfo: UNAVAILABLE
01/22 10:11:44 [RenderSystem] OpenGL using transform constant buffer: false
01/22 10:11:44 [RenderSystem] Vulkan missing requested extension 'VK_KHR_surface'.
01/22 10:11:44 [RenderSystem] Vulkan missing requested extension 'VK_KHR_xcb_surface'.
01/22 10:11:44 [RenderSystem] Loaded video settings config from 'cfg/video.txt'
01/22 10:11:44 Affinity preferences: 'higher' thread assigned to proc mask 0:0xa (index 1) (forced), next proc mask 0:0x5 (index 0)
01/22 10:11:44 [RenderSystem] GL_VERSION=4.5 (Core Profile) Mesa 19.2.8
01/22 10:11:45 ERROR: HangWatchdogInternal_BeginScope() - Hang watchdog not implemented for this platform!
01/22 10:11:45 ERROR: HangWatchdogInternal_EndScope() - Hang watchdog not implemented for this platform!
01/22 10:11:45 [Filesystem] ---------------
01/22 10:11:45 [Filesystem] Path ID:             File Path:
01/22 10:11:45 [Filesystem] ADDONS               "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota_addons/" 
01/22 10:11:45 [Filesystem] CONTENT              "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/content/dota/" 
01/22 10:11:45 [Filesystem] CONTENT              "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/content/core/" 
01/22 10:11:45 [Filesystem] CONTENTADDONS        "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/content/dota_addons/" 
01/22 10:11:45 [Filesystem] CONTENTROOT          "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/content/" 
01/22 10:11:45 [Filesystem] DEFAULT_WRITE_PATH   "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/pak01.vpk" (vpk) /home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/pak01.vpk
01/22 10:11:45 [Filesystem] DEFAULT_WRITE_PATH   "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/" 
01/22 10:11:45 [Filesystem] EXECUTABLE_PATH      "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/bin/linuxsteamrt64/" 
01/22 10:11:45 [Filesystem] GAME                 "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/pak01.vpk" (vpk) /home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/pak01.vpk
01/22 10:11:45 [Filesystem] GAME                 "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/core/pak01.vpk" (vpk) /home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/core/pak01.vpk
01/22 10:11:45 [Filesystem] GAME                 "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/shaders_pcgl.vpk" (vpk) /home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/shaders_pcgl.vpk
01/22 10:11:45 [Filesystem] GAME                 "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/core/shaders_pcgl.vpk" (vpk) /home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/core/shaders_pcgl.vpk
01/22 10:11:45 [Filesystem] GAME                 "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/" 
01/22 10:11:45 [Filesystem] GAME                 "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/core/" 
01/22 10:11:45 [Filesystem] GAMEBIN              "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/bin/linuxsteamrt64/" 
01/22 10:11:45 [Filesystem] GAMEBIN              "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/bin/" 
01/22 10:11:45 [Filesystem] GAMEBIN              "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/core/bin/linuxsteamrt64/" 
01/22 10:11:45 [Filesystem] GAMEBIN              "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/core/bin/" 
01/22 10:11:45 [Filesystem] GAMEROOT             "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/" 
01/22 10:11:45 [Filesystem] MOD                  "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/pak01.vpk" (vpk) /home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/pak01.vpk
01/22 10:11:45 [Filesystem] MOD                  "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/" 
01/22 10:11:45 [Filesystem] PLATFORM             "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/core/pak01.vpk" (vpk) /home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/core/pak01.vpk
01/22 10:11:45 [Filesystem] PLATFORM             "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/core/" 
01/22 10:11:45 [Filesystem] SHADER_SOURCE        "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/src/shaders/dota/" 
01/22 10:11:45 [Filesystem] SHADER_SOURCE        "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/src/shaders/core/" 
01/22 10:11:45 [Filesystem] SHADER_SOURCE        "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/src/shaders/dota_*LANGUAGE*_addons/" 
01/22 10:11:45 [Filesystem] SHADER_SOURCE        "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/src/shaders/dota_addons/" 
01/22 10:11:45 [Filesystem] SHADER_SOURCE        "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/src/shaders/dota_core/" 
01/22 10:11:45 [Filesystem] SHADER_SOURCE_MOD    "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/src/shaders/dota/" 
01/22 10:11:45 [Filesystem] SHADER_SOURCE_ROOT   "/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/src/shaders/" 
01/22 10:11:45 [CommandLine] command line arguments:
01/22 10:11:45 [CommandLine] -con_logfile
01/22 10:11:45 [SteamNetSockets] Initialized low level socket/threading support.
01/22 10:11:45 [SteamNetSockets] Set SteamNetworkingSockets P2P_STUN_ServerList to '153.254.86.181:3478' as per SteamNetworkingSocketsSerialized
01/22 10:11:45 [SteamNetSockets] SDR RelayNetworkStatus:  avail=Attempting  config=Attempting  anyrelay=Not Attempted   (Attempt #1 to fetch config from https://api.steampowered.com/ISteamApps/GetSDRConfig/v1?appid=570&partner=valve)
01/22 10:11:45 [SteamNetSockets] AuthStatus (steamid:76561199053142417):  Attempting  (Requesting cert)
01/22 10:11:45 [SteamNetSockets] Pending ping measurement until network config is obtained.
01/22 10:11:45 [SteamNetSockets] SDR RelayNetworkStatus:  avail=Attempting  config=Attempting  anyrelay=Waiting   (Attempt #1 to fetch config from https://api.steampowered.com/ISteamApps/GetSDRConfig/v1?appid=570&partner=valve)
01/22 10:11:45 [Networking] Got ISteamNetworkingSockets user interfaces from standalone lib
01/22 10:11:45 [Networking] Network System Initialized
01/22 10:11:46 ConVarRef r_dota_color_correction doesn't point to an existing ConVar
01/22 10:11:46 ConVarRef r_dota_fxaa doesn't point to an existing ConVar
01/22 10:11:46 Physics Console Communications is not initialized
01/22 10:11:46 [SoundSystem] Sound Initialization: Finish, Sampling Rate: 44100
01/22 10:11:46 [Localization System] PERF WARNING: 'resource/keybindings_english.txt' is in UTF-16 format, converting to UTF-8 will allow faster loading of it
01/22 10:11:46 Affinity preferences: no preferences, skipping assignment
01/22 10:11:47 WARNING: CDirWatcher not implemented
01/22 10:11:47  failed to dlopen cairo.so error=cairo.so: cannot open shared object file: No such file or directory
01/22 10:11:47 WARNING: CDirWatcher not implemented
01/22 10:11:47 ILocalize::AddFile() failed to load file "resource/dota_english.txt".
01/22 10:11:47 [SceneFileCache] Loaded 71886 .vcds from 187 vcdlist files
01/22 10:11:47 [Client] Event System loaded 96 events from file: vpk:/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/core/pak01.vpk:resource/core.gameevents.
01/22 10:11:47 [Client] Event System loaded 258 events from file: vpk:/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/pak01.vpk:resource/game.gameevents.
01/22 10:11:47 [Client] Event System loaded 1 events from file: vpk:/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/pak01.vpk:./resource/port.gameevents.
01/22 10:11:47 ReadSteamRemoteStorageFile( bufOut, "cfg/pending_replay_requests.lst" ) -> 0.000318 seconds
01/22 10:11:48 ReadSteamRemoteStorageFile( bufOut, "cfg/dotakeys_personal.lst" ) -> 0.000565 seconds
01/22 10:11:48 ReadSteamRemoteStorageFile( bufOut, "scripts/clientstorage.txt" ) -> 0.000290 seconds
01/22 10:11:48 [Networking] CEntitySystem::BuildEntityNetworking (parallel build of client) took 409.144 msecs for 1210 out of 1231 classes
01/22 10:11:51 Inventory shuffle information file "cfg/dota_player_loadout_shuffle.txt" not found in remote storage; not loading.
01/22 10:11:51 Telling Steam it is safe to update the app
01/22 10:11:51 [GCClient] Logged on to Steam, set GC connection state to NO_SESSION
01/22 10:11:52 [Server] Event System loaded 96 events from file: vpk:/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/core/pak01.vpk:resource/core.gameevents.
01/22 10:11:52 [Server] Event System loaded 258 events from file: vpk:/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/pak01.vpk:resource/game.gameevents.
01/22 10:11:52 [Server] Event System loaded 1 events from file: vpk:/home/dunadee/.local/share/Steam/steamapps/common/dota 2 beta/game/dota/pak01.vpk:./resource/port.gameevents.
01/22 10:11:52  failed to dlopen libtensorflow.so error=libtensorflow.so: cannot open shared object file: No such file or directory
01/22 10:11:53 [Networking] CEntitySystem::BuildEntityNetworking (parallel build of server) took 472.913 msecs for 1208 out of 1227 classes
01/22 10:11:58 Failed to load mutations file! scripts/battlepass/ti2018/mutations.txt
01/22 10:11:58 [Vfx] No valid vcs file found for shader generic.vfx
01/22 10:11:58 [MaterialSystem] material2.cpp(1239): Error creating shader generic.vfx for material dev/panorama_world_panel_lit_default.vmat!
01/22 10:11:58 [MaterialSystem] material2.cpp(1239): Error creating shader generic.vfx for material dev/panorama_world_panel_lit_default.vmat!
01/22 10:11:58 [MaterialSystem] material2.cpp(1239): Error creating shader generic.vfx for material dev/panorama_world_panel_lit_default.vmat!
01/22 10:11:58 [MaterialSystem] material2.cpp(1239): Error creating shader generic.vfx for material dev/panorama_world_panel_lit_default.vmat!
01/22 10:11:58 [MaterialSystem] material2.cpp(1239): Error creating shader generic.vfx for material dev/panorama_world_panel_lit_opaque.vmat!
01/22 10:11:58 [MaterialSystem] material2.cpp(1239): Error creating shader generic.vfx for material dev/panorama_world_panel_lit_opaque.vmat!
01/22 10:11:58 [MaterialSystem] material2.cpp(1239): Error creating shader generic.vfx for material dev/panorama_world_panel_lit_opaque.vmat!
01/22 10:11:58 [MaterialSystem] material2.cpp(1239): Error creating shader generic.vfx for material dev/panorama_world_panel_lit_opaque.vmat!
01/22 10:11:58 [ResourceSystem] Error loading resource file "soundevents/soundevents_test.vsndevts_c" (Error: ERROR_FILEOPEN)
01/22 10:11:58 [SplitScreen] Loaded 'user.vcfg' from SteamRemoteStorage, 876 bytes OK
01/22 10:11:58 name[ 0 ] changing from 'unnamed' to 'tobykl'
01/22 10:11:58 name[ 0 ] changing from 'tobykl' to 'tobykl'
01/22 10:11:58 [SplitScreen] Loaded 'user_keys.vcfg' from SteamRemoteStorage, 54 bytes OK
01/22 10:11:58 [HostStateManager] CHostStateMgr::QueueNewRequest( Idle (levelload), 1 )
01/22 10:11:58 ChangeGameUIState: DOTA_GAME_UI_STATE_INVALID -> DOTA_GAME_UI_STATE_LOADING_SCREEN
01/22 10:11:58 Source2Init OK
01/22 10:11:58 [EngineServiceManager] SwitchToLoop levelload requested:  id [1] addons []
01/22 10:11:58 [ResourceSystem] Error loading resource file "panorama/layout/worldui_text_subtitle.vxml_c" (Error: ERROR_FILEOPEN)
01/22 10:11:58 WARNING: CDirWatcher not implemented
01/22 10:11:58 ReadSteamRemoteStorageFile( bufOut, "cfg/viewedheropatches.cfg" ) -> 0.000340 seconds
01/22 10:11:58 ReadSteamRemoteStorageFile( bufOut, "cfg/challenge_selections.cfg" ) -> 0.000203 seconds
01/22 10:11:58 WriteSteamRemoteStorageFileAsync( "cfg/last_match.dat" ) -> at 14.432
01/22 10:11:58 ReadSteamRemoteStorageFile( bufOut, "scripts/dota_acknowledged_violators.txt" ) -> 0.000338 seconds
01/22 10:11:58 ReadSteamRemoteStorageFile( bufOut, "cfg/herobuilds.cfg" ) -> 0.000294 seconds
01/22 10:11:58 ReadSteamRemoteStorageFile( bufOut, "cfg/dismissed_help_tips.txt" ) -> 0.000240 seconds
01/22 10:11:59 ReadSteamRemoteStorageFile( bufOut, "scripts/lobby_settings.txt" ) -> 0.000330 seconds
01/22 10:11:59 ILocalize::AddFile() failed to load file "resource/localization/broadcastfacts_english.txt".
01/22 10:11:59 [HostStateManager] Idle (levelload)
01/22 10:11:59 [SplitScreen] Writing configuration for slot 0
01/22 10:11:59 [SplitScreen] Saved 'user_keys.vcfg' to SteamRemoteStorage, 54 bytes OK
01/22 10:11:59 [SplitScreen] Saved user config file 'cfg/user_keys_1092876689_slot0.vcfg'
01/22 10:11:59 [SplitScreen] Saved offline user config file 'cfg/user_keys_0_slot0.vcfg'
01/22 10:11:59 [SplitScreen] Saved 'user.vcfg' to SteamRemoteStorage, 876 bytes OK
01/22 10:11:59 [SplitScreen] Saved user config file 'cfg/user_1092876689_slot0.vcfg'
01/22 10:11:59 [SplitScreen] Saved offline user config file 'cfg/user_0_slot0.vcfg'
01/22 10:11:59 [Steam] Running SteamAPI_RunCallbacks for first time
01/22 10:11:59 [SteamNetSockets] Got SDR network config.  Loaded revision 293 OK
01/22 10:11:59 [SteamNetSockets] Performing ping measurement
01/22 10:11:59 [SteamNetSockets] SDR RelayNetworkStatus:  avail=Attempting  config=OK  anyrelay=Attempting   (Performing ping measurement)
01/22 10:11:59 [SteamNetSockets] AuthStatus (steamid:76561199053142417):  OK  (OK)
01/22 10:11:59 [GCClient] Send msg 4006 (k_EMsgGCClientHello), 135 bytes
01/22 10:11:59 CAsyncWriteInProgress::OnComplete( "cfg/last_match.dat" ) -> Success at 15.226
01/22 10:11:59 [Client] CL:  CLoopModeLevelLoad::MaybeSwitchToGameLoop switching to "game" loopmode with addons ()
01/22 10:11:59 [EngineServiceManager] SwitchToLoop game requested:  id [1] addons []
01/22 10:11:59 [Workshop] Requesting page 1 of items
01/22 10:11:59 [Workshop] Requesting page 1 of items
01/22 10:11:59 [Workshop] Requesting page 1 of items
01/22 10:11:59 ReadSteamRemoteStorageFile( bufOut, "cfg/chat.cfg" ) -> 0.000708 seconds
01/22 10:11:59 [Panorama] DispatchAsyncEvent backlog, failed to dispatch all this frame. Remaining queue depth: 15861 (was 17591)
01/22 10:11:59 Affinity preferences: no preferences, skipping assignment
01/22 10:11:59 [Server] CNetworkGameServerBase::SetServerState (ss_dead -> ss_waitingforgamesessionmanifest)
01/22 10:11:59 [Server] SV:  maxplayers set to 1
01/22 10:11:59 Initializing script VM...
01/22 10:11:59 ...done
01/22 10:11:59 CSpeechScriptBridge initializing...
01/22 10:11:59 [ResourceSystem] Error loading resource file "scripts/misc.vdata_c" (Error: ERROR_FILEOPEN)
01/22 10:11:59 [Server] SV:  Spawn Server: <empty>
01/22 10:12:00 [Server] CNetworkGameServerBase::SetServerState (ss_waitingforgamesessionmanifest -> ss_loading)
01/22 10:12:00 [Panorama] DispatchAsyncEvent backlog, failed to dispatch all this frame. Remaining queue depth: 17276 (was 18996)
01/22 10:12:00 **** Panel TimeScrubSlider has fill-parent-flow for width, but isn't in a flowing right layout
01/22 10:12:00 [Server] CNetworkGameServerBase::SetServerState (ss_loading -> ss_active)
01/22 10:12:00 [GCClient] Trying to send message type 7387 (k_EMsgDOTAGetEventPoints) before GC connection established.  This is guaranteed to fail.
01/22 10:12:00 [Panorama] DispatchAsyncEvent backlog, failed to dispatch all this frame. Remaining queue depth: 12532 (was 17282)
01/22 10:12:00 [Client] CL:  CWaitForGameServerStartupPrerequisite done waiting for server
01/22 10:12:00 [Client] CL:  CCreateGameClientJob creating client connection to 'loopback'
01/22 10:12:00 [SteamNetSockets] [#3379640783 pipe] connected
01/22 10:12:00 [SteamNetSockets] [#3838243856 pipe] connected
01/22 10:12:00 [Networking] Created poll group for socket 'client'
01/22 10:12:00 [Networking] Created poll group for socket 'server'
01/22 10:12:00 [Networking] Connected loopback client=c97139cf@loopback:1 <-> server=e4c6f410@loopback:0
01/22 10:12:00 [GCClient] Trying to send message type 7427 (k_EMsgGCNotificationsRequest) before GC connection established.  This is guaranteed to fail.
01/22 10:12:00 [Panorama] DispatchAsyncEvent backlog, failed to dispatch all this frame. Remaining queue depth: 9011 (was 12631)
01/22 10:12:00 [NetSteamConn] Associating NetChan       tobykl            loopback:0[1] (loopback:0) with Steam Net Connection handle #3838243856 (userdata 0)
01/22 10:12:00 [Panorama] DispatchAsyncEvent backlog, failed to dispatch all this frame. Remaining queue depth: 6074 (was 11334)
01/22 10:12:00 [SteamNetSockets] Relay iad#260 (162.254.192.99:27038) is going offline in 293 seconds
01/22 10:12:00 [NetSteamConn] Associating NetChan       client            loopback:1[0] (loopback:1) with Steam Net Connection handle #3379640783 (userdata 1)
01/22 10:12:00 [Client] CL:  Connected to 'loopback:1'
01/22 10:12:00 [GCClient] Recv msg 4009 (k_EMsgGCClientConnectionStatus), 12 bytes
01/22 10:12:00 [GCClient] In logon queue; waiting for GC to confirm connection.
01/22 10:12:00 [Server] SV:  Sending server info to client 'tobykl' at loopback:0
01/22 10:12:00 [Server] SV:  WriteInitialSpawnGroups sending 1 groups
01/22 10:12:00 [Panorama] DispatchAsyncEvent backlog, failed to dispatch all this frame. Remaining queue depth: 2059 (was 6189)
01/22 10:12:00 [Client] Game: "Dota 2"
01/22 10:12:00 [Client] Map: "<empty>"
01/22 10:12:00 [Client] Players: 1 (0 bots) / 1 humans
01/22 10:12:00 [Client] Build: 8852 (revision 6323407)
01/22 10:12:00 [Client] Server Number: 1
01/22 10:12:00 [SignonState] CL:  CNetworkGameClient::ProcessServerInfo
01/22 10:12:00 ReadSteamRemoteStorageFile( bufOut, "voice_ban.dt" ) -> 0.000447 seconds
01/22 10:12:00 [RESPONSE_RULES] scripts/talker/response_rules_antimage.txt(token 72) : Multiple definitions for criteria 'isweiloadoutspawn' [627352332]
01/22 10:12:01 [RESPONSE_RULES] CResponseSystem:  scripts/talker/response_rules.txt (25421 rules, 2600 criteria, and 25421 responses)
01/22 10:12:01 WARNING: unable to link r_camerapos and r_camerapos because both ConCommands are not marked FCVAR_LINKED_CONCOMMAND
01/22 10:12:01 WARNING: unable to link r_incrementlodscale and r_incrementlodscale because both ConCommands are not marked FCVAR_LINKED_CONCOMMAND
01/22 10:12:01 WARNING: unable to link r_setpos and r_setpos because both ConCommands are not marked FCVAR_LINKED_CONCOMMAND
01/22 10:12:01 WARNING: unable to link r_entpos and r_entpos because both ConCommands are not marked FCVAR_LINKED_CONCOMMAND
01/22 10:12:01 WARNING: unable to link r_cubemap_takesnapshot and r_cubemap_takesnapshot because both ConCommands are not marked FCVAR_LINKED_CONCOMMAND
01/22 10:12:01 [GCClient] Recv msg 7388 (k_EMsgDOTAGetEventPointsResponse), 321 bytes
01/22 10:12:01 [GCClient] Recv msg 4004 (k_EMsgGCClientWelcome), 526 bytes
01/22 10:12:01 [GCClient] Received client welcome from GC.
01/22 10:12:01 [SOCache] CMsgClientWelcome subscribed: 1 up-to-date, 0 out-of-date
01/22 10:12:01 [GCClient] Recv msg 29 (k_ESOMsg_CacheSubscribedUpToDate), 44 bytes
01/22 10:12:01 [GCClient] Send msg 7197 (k_EMsgGCMatchmakingStatsRequest), 8 bytes
01/22 10:12:02 [GCClient] Recv msg 8747 (k_EMsgGCToClientGuildMembersDataUpdated), 34 bytes
01/22 10:12:02 [GCClient] Recv msg 8678 (k_EMsgGCToClientGuildMembershipUpdated), 14 bytes
01/22 10:12:02 [GCClient] Recv msg 7198 (k_EMsgGCMatchmakingStatsResponse), 323 bytes
01/22 10:12:05 [GCClient] Recv msg 8675 (k_EMsgGCToClientGuildDataUpdated), 1227 bytes
01/22 10:12:05 [GCClient] Recv msg 8689 (k_EMsgGCToClientAccountGuildEventDataUpdated), 51 bytes
01/22 10:12:11 [SteamNetSockets] Ping measurement completed
01/22 10:12:11 [SteamNetSockets] Ping location: hkg=25+2,sgp=44+4/45+4,tyo=55+5,tyo1=60+6/56+5,maa=431+43/78+4,bom=341+34/102+4,dxb=236+23/128+4,lax=154+15,sea=155+15,iad=207+20/204+18,fra=208+20,gru=334+33/328+17
01/22 10:12:11 [SteamNetSockets] SDR RelayNetworkStatus:  avail=OK  config=OK  anyrelay=OK   (OK)
01/22 10:12:12 Overflow of the sheet cache texture. Releasing and starting over. This is ok in asset browser, not so ok if the game is triggering this!
01/22 10:12:13 [Client] CL:  CGameClientConnectPrerequisite connection succeeded
01/22 10:12:13 ChangeGameUIState: DOTA_GAME_UI_STATE_LOADING_SCREEN -> DOTA_GAME_UI_STATE_DASHBOARD
01/22 10:12:13 [Server] SV:  IGameSystem::LoopActivateAllSystems {
01/22 10:12:13 [Host] HO:  IGameSystem::LoopActivateAllSystems {
01/22 10:12:13 [Host] HO:  } IGameSystem::LoopActivateAllSystems done
01/22 10:12:13 [Server] SV:  } IGameSystem::LoopActivateAllSystems done
01/22 10:12:13 [Server] SV:  Game started
01/22 10:12:13 [Client] CL:  IGameSystem::LoopActivateAllSystems {
01/22 10:12:13 ReadSteamRemoteStorageFile( bufOut, "scripts/control_groups.txt" ) -> 0.000436 seconds
01/22 10:12:13 [Client] CL:  } IGameSystem::LoopActivateAllSystems done
01/22 10:12:13 [SignonState] CL:  CNetworkGameClient::OnSwitchLoopModeFinished( game : success )
01/22 10:12:13 GC reported matchgroups version 107
01/22 10:12:13 [WeekendTourney] Received weekend tourney schedule.
01/22 10:12:13 [GCClient] Send msg 2536 (k_EMsgGCRequestStoreSalesData), 12 bytes
01/22 10:12:13 [Server] SV:  Sending full update to client tobykl (reason:  initial update)
01/22 10:12:13 [Client] CL:  Receiving uncompressed update from server
01/22 10:12:13 [Networking]       client            loopback:1[0]:  NetChan Setting Timeout to 30.00 seconds
01/22 10:12:13 [Client] CL:  Signon traffic "client":  incoming 59.845 KB [393 pkts], outgoing 2.390 KB [389 pkts]
01/22 10:12:13 [Entity System] ExecuteQueuedCreation() took 1072us [ Spawned: 1, Activated 1, PostDataUpdate: 0, Destroyed: 0, Deallocated: 0, PvsUpdate: 0 ]
01/22 10:12:13 ConVarRef dota_camera_toggle_select_to_follow doesn't point to an existing ConVar
01/22 10:12:13 Failed to load inventory scratchpad information file "cfg/dota_player_scratchpad.txt"!
01/22 10:12:13 [Networking]       tobykl            loopback:0[1]:  NetChan Setting Timeout to 20.00 seconds
01/22 10:12:13 [GCClient] Send msg 7009 (k_EMsgGCJoinChatChannel), 23 bytes
01/22 10:12:13 [GCClient] Send msg 7009 (k_EMsgGCJoinChatChannel), 23 bytes
01/22 10:12:13 [GCClient] Send msg 7009 (k_EMsgGCJoinChatChannel), 25 bytes
01/22 10:12:13 [GCClient] Send msg 7009 (k_EMsgGCJoinChatChannel), 27 bytes
01/22 10:12:13 [GCClient] Send msg 7009 (k_EMsgGCJoinChatChannel), 28 bytes
01/22 10:12:13 Job CDOTATeamsController::Initialize has spent 3.000ms without yielding: 4.390ms
01/22 10:12:13 [WeekendTourney] Auto selecting weekend tourney division 'se_asia' based on ping to best region of 45ms
01/22 10:12:13 [WeekendTourney] Selected division changed to 'se_asia'
01/22 10:12:13 [GCClient] Recv msg 2538 (k_EMsgGCRequestStoreSalesDataUpToDateResponse), 16 bytes
01/22 10:12:13 [GCClient] Recv msg 7010 (k_EMsgGCJoinChatChannelResponse), 45 bytes
01/22 10:12:13 [GCClient] Recv msg 7010 (k_EMsgGCJoinChatChannelResponse), 45 bytes
01/22 10:12:13 [GCClient] Recv msg 7010 (k_EMsgGCJoinChatChannelResponse), 47 bytes
01/22 10:12:13 [GCClient] Recv msg 7010 (k_EMsgGCJoinChatChannelResponse), 31 bytes
01/22 10:12:13 [GCClient] Recv msg 7010 (k_EMsgGCJoinChatChannelResponse), 55 bytes
01/22 10:12:13 [GCClient] Recv msg 7010 (k_EMsgGCJoinChatChannelResponse), 32 bytes
01/22 10:12:13 [GCClient] Recv msg 7010 (k_EMsgGCJoinChatChannelResponse), 56 bytes
01/22 10:12:14 Job ShowIntroMovie has spent 3.000ms without yielding: 10.461ms
01/22 10:12:14 [GCClient] Send msg 7387 (k_EMsgDOTAGetEventPoints), 25 bytes
01/22 10:12:14 [Panorama] DispatchAsyncEvent backlog, failed to dispatch all this frame. Remaining queue depth: 254 (was 414)
01/22 10:12:14 [GCClient] Send msg 7427 (k_EMsgGCNotificationsRequest), 17 bytes
01/22 10:12:14 [GCClient] Send msg 7503 (k_EMsgClientToGCEmoticonDataRequest), 17 bytes
01/22 10:12:14 Job CAsyncDataCache::RequestCurrentData has spent 3.000ms without yielding: 8.179ms
01/22 10:12:14 [GCClient] Send msg 8687 (k_EMsgClientToGCRequestAccountGuildEventData), 23 bytes
01/22 10:12:14 [GCClient] Send msg 8673 (k_EMsgClientToGCRequestGuildData), 21 bytes
01/22 10:12:14 [GCClient] Send msg 8713 (k_EMsgClientToGCRequestActiveGuildChallenge), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 8137 (k_EMsgClientToGCMyTeamInfoRequest), 17 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Recv msg 7388 (k_EMsgDOTAGetEventPointsResponse), 328 bytes
01/22 10:12:14 [GCClient] Recv msg 7428 (k_EMsgGCNotificationsResponse), 19 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 Job CAsyncDataCache::RequestCurrentData has spent 3.000ms without yielding: 3.049ms
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Recv msg 7504 (k_EMsgGCToClientEmoticonData), 59 bytes
01/22 10:12:14 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 50 bytes
01/22 10:12:14 [GCClient] Recv msg 8688 (k_EMsgClientToGCRequestAccountGuildEventDataResponse), 55 bytes
01/22 10:12:14 [GCClient] Recv msg 8674 (k_EMsgClientToGCRequestGuildDataResponse), 1236 bytes
01/22 10:12:14 [GCClient] Recv msg 8714 (k_EMsgClientToGCRequestActiveGuildChallengeResponse), 39 bytes
01/22 10:12:14 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 567 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 23 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 [GCClient] Send msg 7534 (k_EMsgClientToGCGetProfileCard), 22 bytes
01/22 10:12:14 WriteSteamRemoteStorageFileAsync( "cfg/chat.cfg" ) -> at 30.289
01/22 10:12:14 WriteSteamRemoteStorageFileAsync( "cfg/chat.cfg" ) -> at 30.330
01/22 10:12:14 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 105 bytes
01/22 10:12:14 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 38 bytes
01/22 10:12:14 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 62 bytes
01/22 10:12:14 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 168 bytes
01/22 10:12:14 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 98 bytes
01/22 10:12:14 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 55 bytes
01/22 10:12:14 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 91 bytes
01/22 10:12:14 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 64 bytes
01/22 10:12:14 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 39 bytes
01/22 10:12:14 CAsyncWriteInProgress::OnComplete( "cfg/chat.cfg" ) -> Success at 30.366
01/22 10:12:14 CAsyncWriteInProgress::OnComplete( "cfg/chat.cfg" ) -> Success at 30.367
01/22 10:12:14 WriteSteamRemoteStorageFileAsync( "cfg/chat.cfg" ) -> at 30.379
01/22 10:12:14 WriteSteamRemoteStorageFileAsync( "cfg/chat.cfg" ) -> at 30.413
01/22 10:12:14 [GCClient] Send msg 8729 (k_EMsgClientToGCRequestAccountGuildPersonaInfoBatch), 92 bytes
01/22 10:12:15 CAsyncWriteInProgress::OnComplete( "cfg/chat.cfg" ) -> Success at 30.472
01/22 10:12:15 CAsyncWriteInProgress::OnComplete( "cfg/chat.cfg" ) -> Error (0:37) at 30.581
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 90 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 792 bytes
01/22 10:12:15 [GCClient] Recv msg 8136 (k_EMsgGCToClientTeamsInfo), 17 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 43 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 41 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 69 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 64 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 102 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 69 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 169 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 38 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 299 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 38 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 211 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 197 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 91 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 38 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 89 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 68 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 269 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 386 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 95 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 50 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 95 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 43 bytes
01/22 10:12:15 [GCClient] Recv msg 7535 (k_EMsgClientToGCGetProfileCardResponse), 43 bytes
01/22 10:12:15 [GCClient] Recv msg 8730 (k_EMsgClientToGCRequestAccountGuildPersonaInfoBatchResponse), 149 bytes
01/22 10:12:19 [WorldRenderer] Long grass load frame: 17.497475 ms for 2 grass tiles
01/22 10:12:19 [WorldRenderer] CWorldRendererMgr::ServiceWorldRequests long frame: 17.643344ms
01/22 10:12:20 [GCClient] Send msg 8673 (k_EMsgClientToGCRequestGuildData), 19 bytes
01/22 10:12:20 Job CAsyncDataCache::RequestCurrentData has spent 3.000ms without yielding: 7.239ms
01/22 10:12:20 [Entity System] ExecuteQueuedCreation() took 10184us [ Spawned: 137, Activated 137, PostDataUpdate: 0, Destroyed: 0, Deallocated: 0, PvsUpdate: 0 ]
01/22 10:12:20 [GCClient] Recv msg 8674 (k_EMsgClientToGCRequestGuildDataResponse), 19 bytes
01/22 10:12:20 [Entity System] ExecuteQueuedCreation() took 20144us [ Spawned: 1, Activated 1, PostDataUpdate: 0, Destroyed: 0, Deallocated: 0, PvsUpdate: 0 ]
01/22 10:12:21 [ResourceSystem] Error loading resource file "panorama/images/spellicons/dota_base_ability_png.vtex_c" (Error: ERROR_FILEOPEN)
01/22 10:12:30 [GCClient] Timed out waiting 10 secs for reply msg 7535 from GC.  Reply was not marked critical.
01/22 10:12:30 [GCClient] Timed out waiting 10 secs for reply msg 7535 from GC.  Reply was not marked critical.
01/22 10:12:30 [GCClient] Timed out waiting 10 secs for reply msg 7535 from GC.  Reply was not marked critical.
01/22 10:12:30 [GCClient] Timed out waiting 10 secs for reply msg 7535 from GC.  Reply was not marked critical.
01/22 10:12:30 [GCClient] Timed out waiting 10 secs for reply msg 7535 from GC.  Reply was not marked critical.
01/22 10:12:30 [GCClient] Timed out waiting 10 secs for reply msg 7535 from GC.  Reply was not marked critical.
01/22 10:12:30 [GCClient] Timed out waiting 10 secs for reply msg 7535 from GC.  Reply was not marked critical.
01/22 10:12:30 [GCClient] Timed out waiting 10 secs for reply msg 7535 from GC.  Reply was not marked critical.
01/22 10:12:30 [GCClient] Timed out waiting 10 secs for reply msg 7535 from GC.  Reply was not marked critical.
01/22 10:12:45 [GCClient] Send msg 8303 (k_EMsgClientToGCRequestSocialFeed), 25 bytes
01/22 10:12:46 [GCClient] Recv msg 8304 (k_EMsgClientToGCRequestSocialFeedResponse), 776 bytes
01/22 10:12:47 ProtoBufHelper::ParseJSON: Ignoring extra JSON key 'success', which is not found in protobuf message 'CMsgDOTALeagueList'!
01/22 10:12:47 ProtoBufHelper::ParseJSON: Ignoring extra JSON key 'success', which is not found in protobuf message 'CMsgDOTAPlayerInfoList'!
01/22 10:12:51 [GCClient] Recv msg 7273 (k_EMsgGCChatMessage), 217 bytes
01/22 10:12:57 [SteamNetSockets] Waited 4.6ms for SteamNetworkingSockets lock [GetQuickConnectionStatus]
01/22 10:13:01 [GCClient] Send msg 8727 (k_EMsgClientToGCRequestAccountGuildPersonaInfo), 22 bytes
01/22 10:13:01 [GCClient] Recv msg 8728 (k_EMsgClientToGCRequestAccountGuildPersonaInfoResponse), 21 bytes
01/22 10:13:02 [GCClient] Send msg 7197 (k_EMsgGCMatchmakingStatsRequest), 8 bytes
01/22 10:13:02 [GCClient] Recv msg 7198 (k_EMsgGCMatchmakingStatsResponse), 322 bytes
01/22 10:13:17 ProtoBufHelper::ParseJSON: Ignoring extra JSON key 'success', which is not found in protobuf message 'CMsgDOTAPlayerInfoList'!
01/22 10:13:31 Job CAsyncDataCache::RequestCurrentData has spent 3.000ms without yielding: 3.215ms
01/22 10:13:32 ProtoBufHelper::ParseJSON: Ignoring extra JSON key 'success', which is not found in protobuf message 'CMsgDOTALeagueList'!
01/22 10:13:48 [GCClient] Recv msg 7273 (k_EMsgGCChatMessage), 37 bytes
01/22 10:14:01 Job CAsyncDataCache::RequestCurrentData has spent 3.000ms without yielding: 3.567ms
01/22 10:14:03 [GCClient] Send msg 7197 (k_EMsgGCMatchmakingStatsRequest), 8 bytes
01/22 10:14:03 [GCClient] Recv msg 7198 (k_EMsgGCMatchmakingStatsResponse), 322 bytes
01/22 10:14:04 [GCClient] Recv msg 7273 (k_EMsgGCChatMessage), 35 bytes
```

</p></details>

**Comment by randolphpebenito** [comment]
Got it working. Adjusted the video settings. 

Thanks gazillion times!

**Comment by astaruch** [comment]
Hi,
I'm having the exact same issue. OS: Ubuntu 18.04 (Dell XPS 13 7390), latest updates installed.

I've discovered this issue when I was looking for Vulkan problems, since from this week when I start Dota from the Steam, I'm always getting Windowed mode with OpenGL.

```
<>@as-xps13:~/.steam/steam/steamapps/common/dota 2 beta/game$ ~/.steam/ubuntu12_32/steam-runtime/run.sh ./dota.sh -con_logfile
Using breakpad crash handler
[S_API] SteamAPI_Init(): SteamAPI_IsSteamRunning() did not locate a running instance of Steam.
[S_API] SteamAPI_Init(): Loaded '/home/<>/.steam/linux64/steamclient.so' OK.
[S_API FAIL] SteamAPI_Init() failed; create pipe failed.Setting breakpad minidump AppID = 373300
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
[S_API] SteamAPI_Init(): SteamAPI_IsSteamRunning() did not locate a running instance of Steam.
[S_API] SteamAPI_Init(): Loaded '/home/<>/.steam/linux64/steamclient.so' OK.
[S_API FAIL] SteamAPI_Init() failed; create pipe failed.Attempting to create OpenGL context: v3.3 (Core profile)...success. Checking entry points...success.
Attempting to create OpenGL context: v3.3 (Core profile)...success. Checking entry points...success.
DumpContextInfo: OpenGL vendor Intel Open Source Technology Center
DumpContextInfo: OpenGL renderer Mesa DRI Intel(R) UHD Graphics (CML GT2)
DumpContextInfo: Using OpenGL context version 4.6
DumpContextInfo: Context supports GLSL version 4.60
Fontconfig warning: "/home/<>/.steam/steam/steamapps/common/dota 2 beta/game/core/panorama/fonts/conf.d/41-repl-os-win.conf", line 148: Having multiple values in <test> isn't supported and may not work as expected
Fontconfig warning: "/home/<>/.steam/steam/steamapps/common/dota 2 beta/game/core/panorama/fonts/conf.d/41-repl-os-win.conf", line 160: Having multiple values in <test> isn't supported and may not work as expected
[S_API FAIL] Tried to access Steam interface SteamUtils010 before SteamAPI_Init succeeded.
 failed to dlopen "cairo.so" error=cairo.so: cannot open shared object file: No such file or directory
[S_API] SteamAPI_Init(): SteamAPI_IsSteamRunning() did not locate a running instance of Steam.
[S_API] SteamAPI_Init(): Loaded '/home/<>/.steam/linux64/steamclient.so' OK.
[S_API FAIL] SteamAPI_Init() failed; create pipe failed.[S_API] SteamAPI_Init(): SteamAPI_IsSteamRunning() did not locate a running instance of Steam.
[S_API] SteamAPI_Init(): Loaded '/home/<>/.steam/linux64/steamclient.so' OK.
[S_API FAIL] SteamAPI_Init() failed; create pipe failed.[S_API FAIL] Tried to access Steam interface STEAMREMOTESTORAGE_INTERFACE_VERSION014 before SteamAPI_Init succeeded.
[S_API] SteamAPI_Init(): SteamAPI_IsSteamRunning() did not locate a running instance of Steam.
[S_API] SteamAPI_Init(): Loaded '/home/<>/.steam/linux64/steamclient.so' OK.
[S_API FAIL] SteamAPI_Init() failed; create pipe failed. failed to dlopen "libtensorflow.so" error=libtensorflow.so: cannot open shared object file: No such file or directory
[S_API] SteamAPI_Init(): SteamAPI_IsSteamRunning() did not locate a running instance of Steam.
[S_API] SteamAPI_Init(): Loaded '/home/<>/.steam/linux64/steamclient.so' OK.
[S_API FAIL] SteamAPI_Init() failed; create pipe failed.[S_API FAIL] Tried to access Steam interface SteamFriends017 before SteamAPI_Init succeeded.
[S_API FAIL] Tried to access Steam interface STEAMREMOTESTORAGE_INTERFACE_VERSION014 before SteamAPI_Init succeeded.
[S_API FAIL] Tried to access Steam interface SteamUtils010 before SteamAPI_Init succeeded.
[S_API FAIL] Tried to access Steam interface SteamUser021 before SteamAPI_Init succeeded.
[S_API FAIL] Tried to access Steam interface STEAMHTTP_INTERFACE_VERSION003 before SteamAPI_Init succeeded.
[S_API FAIL] Tried to access Steam interface STEAMUSERSTATS_INTERFACE_VERSION012 before SteamAPI_Init succeeded.
[S_API FAIL] Tried to access Steam interface SteamUser021 before SteamAPI_Init succeeded.
LuaDbg - Debugging stopped
CSteam3Client::Shutdown
```

I've checked this line:
```
 failed to dlopen "cairo.so" error=cairo.so: cannot open shared object file: No such file or directory
```
and it's weird, since lib is on the system.
```
$ sudo apt-get install libcairo-dev
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'libcairo2-dev' instead of 'libcairo-dev'
libcairo2-dev is already the newest version (1.15.10-2ubuntu0.1).
```

Game is playable under OpenGL, but something went definitely wrong.

**Comment by astaruch** [comment]
Hello, 

I'm extending information.

I've figured out, that I haven't had installed Vulkan drivers - I've just selected `vulkan` in the Settings and didn't check that it is still running on OpenGL. So in the first place, I had not Vulkan version running.

I've fixed this by installing Vulkan drivers [ref](https://steamcommunity.com/app/221410/discussions/6/1742227264188882438/)
```
$ sudo apt-get install libvulkan1 libvulkan1:i386 mesa-vulkan-drivers mesa-vulkan-drivers:i386
```
Then verify that Vulkan is working:
```
$ sudo apt-get install vulkan-utils
$ vuklan-smoketest
914 presents in 5.00252 seconds (FPS: 182.708)
^C
```

After that, I was able to start Dota 2 in Vulkan mode.

Sadly, this has far worse performance on my notebook (Dell XPS 13 7390), so I've switched back to the OpenGL.

What I'd like to point out:
1. Until there was a update, that forces Vulkan on the Linux, I was using happily OpenGL.
2. After this patch, when Vulkan is forced and fails to open, since it's not installed, it will fallback to the OpenGL. Problem is that it is always *Windowed mode* with default video quality settings and resolution. It will not persist a new start

So the solution for me was install Vulkan and add `-gl` to the start commands for Dota 2. Maybe this will help others.

**Comment by ii8** [comment]
I get the same error dialog when attempting to start dota since the recent march 6 update.
Have tried the lauch options it suggests, verifying files, using shadercache and not using it, launching from inside steam and from console, the same error persists in all cases.

Running dota like this:
```
GAME_DEBUGGER=gdb ubuntu12_32/steam-runtime/run.sh steamapps/common/dota\ 2\ beta/game/dota.sh
```
produces the following error(only printed when using the debugger):
```
No display adapters found. Falling back to software rasterizer. (nAdapterCount=0, nAdapter=0)
```
And produces the same dialog as in the screenshot in the first post in this issue.
Notably there are no dlopen errors.

this is the full log:
```
Reading symbols from /home/steam/.local/share/Steam/steamapps/common/dota 2 beta/game/bin/linuxsteamrt64/dota2...
(No debugging symbols found in /home/steam/.local/share/Steam/steamapps/common/dota 2 beta/game/bin/linuxsteamrt64/dota2)
Function "main" not defined.
Make breakpoint pending on future shared library load? (y or [n]) [answered N; input not from terminal]
Setting environment variable "LD_PRELOAD" to null value.
LD_PRELOAD = 
LD_LIBRARY_PATH = /home/steam/.local/share/Steam/steamapps/common/dota 2 beta/game/bin/linuxsteamrt64:/home/steam/.local/share/Steam/ubuntu12_64/steam-runtime-heavy/pinned_libs_32:/home/steam/.local/share/Steam/ubuntu12_64/steam-runtime-heavy/pinned_libs_64:/lib/i386-linux-gnu:/usr/local/lib:/lib/x86_64-linux-gnu:/lib:/home/steam/.local/share/Steam/ubuntu12_64/steam-runtime-heavy/lib/i386-linux-gnu:/home/steam/.local/share/Steam/ubuntu12_64/steam-runtime-heavy/usr/lib/i386-linux-gnu:/home/steam/.local/share/Steam/ubuntu12_64/steam-runtime-heavy/lib/x86_64-linux-gnu:/home/steam/.local/share/Steam/ubuntu12_64/steam-runtime-heavy/usr/lib/x86_64-linux-gnu:/home/steam/.local/share/Steam/ubuntu12_64/steam-runtime-heavy/lib:/home/steam/.local/share/Steam/ubuntu12_64/steam-runtime-heavy/usr/lib
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[Detaching after vfork from child process 13037]
Loaded /home/steam/.local/share/Steam/steamapps/common/dota 2 beta/game/bin/linuxsteamrt64/libengine2.so, got 0x55cdd3bacb00
Steam AppId(570), BreakpadId(373300)
Using breakpad crash handler
[S_API] SteamAPI_Init(): Loaded '/home/steam/.local/share/Steam/linux64/steamclient.so' OK.
Game process added : AppID 570 "", ProcID 13034, IP 0.0.0.0:0
Setting breakpad minidump AppID = 570
Forcing breakpad minidump interfaces to load
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
SteamInternal_SetMinidumpSteamID:  Caching Steam ID:  76561198082821288 [API loaded yes]
SteamInternal_SetMinidumpSteamID:  Setting Steam ID:  76561198082821288
SteamAPI_Init succeeded.  SteamID is [U:1:122555560] (76561198082821288), AppID is 570
Setting breakpad minidump AppID = 373300
CSteam3Client::Activate succeeded.  SteamID is [U:1:122555560] (76561198082821288), AppID is 570
Loaded libSDL2-2.0.so.0, got 0x55cdd40b9500
[New Thread 0x7fb4a87ff640 (LWP 13039)]
[New Thread 0x7fb4a57ff640 (LWP 13040)]
[New Thread 0x7fb4a577e640 (LWP 13041)]
[Detaching after fork from child process 13042]
Loaded /home/steam/.local/share/Steam/steamapps/common/dota 2 beta/game/bin/linuxsteamrt64/libtier0.so, got 0x55cdd3bac000
Visibility enabled.
Loaded /home/steam/.local/share/Steam/steamapps/common/dota 2 beta/game/bin/linuxsteamrt64/libfilesystem_stdio.so, got 0x55cdd3bae680
USRLOCAL path using Steam profile data folder:
/home/steam/.local/share/Steam/userdata/122555560/570/local
Loaded /home/steam/.local/share/Steam/steamapps/common/dota 2 beta/game/bin/linuxsteamrt64/libinputsystem.so, got 0x55cdd3bae100
Loaded /home/steam/.local/share/Steam/steamapps/common/dota 2 beta/game/bin/linuxsteamrt64/liblocalize.so, got 0x55cdd58e5600
Loaded /home/steam/.local/share/Steam/steamapps/common/dota 2 beta/game/bin/linuxsteamrt64/librendersystemvulkan.so, got 0x55cdd58e5080
Loaded /home/steam/.local/share/Steam/steamapps/common/dota 2 beta/game/bin/linuxsteamrt64/libresourcesystem.so, got 0x55cdd58e4b00
Loaded /home/steam/.local/share/Steam/steamapps/common/dota 2 beta/game/bin/linuxsteamrt64/libschemasystem.so, got 0x55cdd58e4580
Enabling instance extension: VK_KHR_get_physical_device_properties2.
[New Thread 0x7fb4a5063640 (LWP 13043)]
[New Thread 0x7fb4a4ccb640 (LWP 13044)]
[New Thread 0x7fb4a49ff640 (LWP 13045)]
No display adapters found. Falling back to software rasterizer. (nAdapterCount=0, nAdapter=0)
[Detaching after fork from child process 13046]
```

Another thing I should maybe add is that launching steam takes a very long time since recently, likewise dota takes a very long time to launch if running it from inside steam, when running it from the console it gets to the dialog fast.
During the delay before dota tries to start a process called `CIPCServer::Thr` is running at 100% cpu usage. Once dota fails to start this process disappears again.

**Comment by kisak-valve** [comment]
Hello @ii8, please open a new issue report. There's years separation from this issue report and it would be better to evaluate your issue from scratch.

Closing as outdated.
